manager/manager/views/config.py

```diff
@@ -142,6 +149,49 @@ def configuration_list(request):
 
     return render(request, "configuration_list.html", context)
 
+# sorting function for configurations
+def apply_sorting(queryset, sort_field, sort_dir):
+    order_prefix = '' if sort_dir == 'asc' else '-'
+    
+    if sort_field == 'min_media':
+        import re
+        configurations_list = list(queryset)
```

Comment:
All this code is overly complicated, unnecessary and under-performing. Use `config.size` instead which is a model field and can thus be handled like the others